src/pages/comparison/components/comparison-element.component.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { Profile } from './profile.component';
+import { Rate } from './rate.component';
+
+import '../comparison.scss';
+
+import { PetToCompare } from '../interfaces/pet.interface';
+
+export const ComparisonElement: React.FC<PetToCompare> = props => {
+  return (
+    <Link className="link-style" to={`/pet/${props.id}`} key={props.id}>
```

Comment:
I would recomment to move `/pet/${props.id}` to the `ROUTES` constants.

For example
```JavaScript
// routes.constants.ts
const ROUTES = {
  pet: (id: number) => `/pet/${id}`,
}

```